Cargo.toml

```diff
@@ -15,15 +15,15 @@ default = ["chrono"]
 chrono = ["dep:chrono"]
 
 [dependencies]
-chrono = { version = "0.4.24", optional = true, default-features = false, features = ["clock"]}
+chrono = { version = "0.4.31", optional = true, default-features = false, features = ["clock"]}
 
 [dev-dependencies]
-arrayvec = "0.7.2"
+arrayvec = "0.7.4"
 assert_matches = "1.5.0"
 dhat = "0.3.2"
 env_logger = "0.10.0"
 is-terminal = "0.4.7"
```

Comment:
Sidenote: can we get rid of `is-terminal`?

Reply:
Ah true!

Reply:
Resolved in https://github.com/tandemdrive/syslog-fmt/pull/23/commits/9208c47d23309f2faaeae32729b1b6d89d893faf